contrib/localnet/orchestrator/start-zetae2e.sh

```diff
@@ -90,9 +90,47 @@ geth --exec "eth.sendTransaction({from: eth.coinbase, to: '${address}', value: w
 address=$(yq -r '.additional_accounts.user_migration.evm_address' config.yml)
 echo "funding migration tester address ${address} with 10000 Ether"
 geth --exec "eth.sendTransaction({from: eth.coinbase, to: '${address}', value: web3.toWei(10000,'ether')})" attach http://eth:8545
-
 ### Run zetae2e command depending on the option passed
 
+# Mode migrate is used to run the e2e tests before and after the TSS migration
+# It runs the e2e tests with the migrate flag which triggers a TSS migration at the end of the tests. Once the migrationis done the first e2e test is complete
+# The second e2e test is run after the migration to ensure the network is still working as expected with the new tss address
+if [ "$LOCALNET_MODE" == "migrate" ]; then
```

Comment:
what does migrate mean in this context? I.e. the name is not descriptive as we have `LOCALNET_MODE=upgrade` which is also a "migration"

Reply:
0e6fa1c995731728ae47b26b1f03a76ae54aa22b